WordPress/twentytwentyfive
# Fine tune "Team photos" pattern

**Description**

I'm updating the images to match the ones in the design, and making the text bigger.

**Screenshots**



Before

https://github.com/user-attachments/assets/2527d044-e3ef-4ab3-8de6-9b942f2d29c9

After

https://github.com/user-attachments/assets/23033edf-b33f-4385-b13b-a5e0e1c5b28e

**Testing Instructions**

1. Create a page.
2. Add the team pattern.
3. Confirm that the images look like the Figma design and that the text is bigger and similar to the one in Figma.
